utilities/version_utils.py

```
@@ -0,0 +1,91 @@
+import pytest
+from typing import Any
+
+LOGGER = get_logger(name=__name__)
```

Ruff (0.11.9)

Undefined name `get_logger`

(F821)